client/client.go

```diff
@@ -963,3 +984,93 @@ func (r *NotaryRepository) DeleteTrustData() error {
 	}
 	return nil
 }
+
+// ListRootCerts returns current trusted root certificates.
+// (In particular, not old certificates which we have already
+// rotated from but are still using for signing to allow rollover,
+// even if they are still trusted locally.)
+func (r *NotaryRepository) ListRootCerts() ([]*x509.Certificate, error) {
+	_, err := r.Update(false)
+	if err != nil {
+		return nil, err
+	}
+
+	keyIDs := r.tufRepo.Root.Signed.Roles[data.CanonicalRootRole].KeyIDs
```

Comment:
Yeah, there are some further changes that need to be made but my idea was that having the root.json is equally good for TOFU because it contains all the certs we would have written to the store.